include/rocksdb/advanced_options.h

```diff
@@ -644,6 +644,16 @@ struct AdvancedColumnFamilyOptions {
   // Dynamically changeable through SetOptions() API
   uint64_t ttl = 0;
 
+  // Files older than this value will be picked up for compaction, and
+  // re-written to the same level as they were before.
+  // Only supported in Level compaction.
+  // Pre-req: max_open_file == -1.
+  // unit: seconds. Ex: 7 days = 7 * 24 * 60 * 60
+  // Default: 0 (disabled)
+  //
+  // Dynamically changeable through SetOptions() API
+  uint64_t periodic_compaction_time = 0;
```

Comment:
Now we suggest that unit is included in the variable name. For example `periodic_compaction_sec`.

Reply:
done. changed to `periodic_compaction_seconds`.